src/ape_init/_cli.py

```diff
@@ -33,6 +33,43 @@ def cli(cli_ctx, github):
             else:
                 folder.mkdir(exist_ok=False)
 
+        body = """
+# User Specified Ignored
+
+
+# Ape stuff
+.build/
+.pytest_cache
+__pycache__
+.env
+.cache
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# pyenv
+.python-version
+
+# Distribution / packaging
+.Python
+build/
+eggs/
+.eggs/
+*.egg-info/
+*.egg
```

Comment:
I think all of this stuff would be user-specific, what they want to ignore or not

Reply:
ill rm 50-65. I want to keep .env and .venv since we recommend not using a .env and .venv